AGENTS.md

```
@@ -0,0 +1,233 @@
+| CLI | пакет `cli` |
+| Логи | `logos` (`oscript.app.opm`) |
+| Unit | `1testrunner` |
+| BDD | `1bdd` (Gherkin на русском) |
+| Coverage | `coverage` + `oscript -codestat=` |
+| CI | GitHub Actions + SonarQube (`sonar.openbsl.ru`) |
+
+Runtime-зависимости — в корневом `packagedef`: `fs`, `asserts`, `fluent`, `logos`, `cli`, `tempfiles`, `gitrunner`, `reflector`.
+
+## Структура репозитория
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)